version-control/command-line/tutorial.md

```diff
@@ -62,6 +62,33 @@ $ git log
 
 > Don't forget, you can hit `q` to quit out of a log if it's too long.
 
+By default, with no arguments, git log shows the most recent commits first.
+
+If we want to limit the number of log entries displayed we can use the a number as an option, such as -3 to show only the last three entries. 
```

Comment:
Small change here too: "...we can use **the a** number as an option..." should read as "we can use a number as an option"

Reply:
Nice spot, thanks @MarkCBJSS